ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Git/CommitsSample.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections.Generic;
+using Microsoft.TeamFoundation.SourceControl.WebApi;
+
+namespace Microsoft.TeamServices.Samples.Client.Git
+{
+    [ClientSample(GitWebApiConstants.AreaName, "commits")]
```

Comment:
No constant for "commits" in the public client library? If not, you should log an internal work item to address this at some point and then loop back and fix the sample. I'd expect a constant in `GitWebApiConstants`